src/wh_client_crypto.c

```diff
@@ -156,6 +157,284 @@ int wh_Client_RngGenerate(whClientContext* ctx, uint8_t* out, uint32_t size)
     return ret;
 }
 
+#ifndef NO_AES
+
+#ifdef HAVE_AES_CBC
+int wh_Client_AesCbc(whClientContext* ctx,
+        Aes* aes, int enc,
```

Comment:
missing NULL check on `Aes* aes`. Pretty sure you only want this function called by the cryptoCb, in which case you know that the calling wolfCrypt AES API does the NULL check, but since it is exposed in a header, we should check it anyway in case we were to want to use it elsewhere

Reply:
updated and added to gem as well.  changed argument order to match wc.